app/robots/iCubGazeboV2_5/oculusConfig.ini

```diff
@@ -17,6 +17,10 @@ enableMoveRobot         1
 # when the human rotates inside the virtualizer
 playerOrientationThreshold    0.2
 
+# For kinematic scaling for task-space retargeting
+humanHeight                   1.76
+robotArmSpan                  1.10
```

Comment:
Is this parameter really necessary? Can we retrieve it from the model? If not, I would add a comment on how to measure this parameter. From which points is it measured?

Reply:
yes, it is really necessary. No, we do not load the model in the process.
It is normally specified in the robot's specifications documents.

Reply:
Ok got it, maybe it is worth adding a link to the documentation where you find it :thinking:

Reply:
`robotArmSpan` is the distance between the two left and right hand palms.

Reply:
Ok got it. I think it may be worth adding this information somewhere in case we need to use a different robot. It should be sufficient just a comment on how to measure it, I guess.

Reply:
I updated the definition of the arm span, and we measured on the real robot.

Reply:
done!